dev/dags/example_duckdb_dag.py

```diff
@@ -0,0 +1,42 @@
+"""
+An example DAG that uses Cosmos to render a dbt-duck project into an Airflow DAG.
+"""
+
+import os
+from datetime import datetime
+from pathlib import Path
+
+from cosmos import DbtDag, ProfileConfig, ProjectConfig
+from cosmos.profiles import DuckDBUserPasswordProfileMapping
+
+DEFAULT_DBT_ROOT_PATH = Path(__file__).parent / "dbt"
+DBT_ROOT_PATH = Path(os.getenv("DBT_ROOT_PATH", DEFAULT_DBT_ROOT_PATH))
+
+profile_config = ProfileConfig(
+    profile_name="default",
+    target_name="dev",
+    profile_mapping=DuckDBUserPasswordProfileMapping(
+        conn_id="duckdb_default",
+        profile_args={"path": "jaffle_shop.duck_db"},
+        disable_event_tracking=True,
+    ),
+)
+
+# [START local_example]
+basic_cosmos_dag = DbtDag(
+    # dbt/cosmos-specific parameters
+    project_config=ProjectConfig(
+        DBT_ROOT_PATH / "jaffle_shop_duckdb",
```

Comment:
How different is this Jaffle shop project from the other two dbt jaffle shop projects we already have in the Cosmos repo:
- https://github.com/astronomer/astronomer-cosmos/tree/main/dev/dags/dbt/jaffle_shop
- https://github.com/astronomer/astronomer-cosmos/tree/main/dev/dags/dbt/altered_jaffle_shop

Would it be possible to use one of the existing dbt projects and use a different `profile.yml`?

Reply:
Yes we can! that's mainly where it differs, makes more sense to reuse what we have.

Reply:
It would be really good if we did not add yet another jaffle shop copy to the Cosmos repo.
If we can reuse one of the existing ones, let's do this.
Otherwise, if we cannot for any reason, we can create a smaller dbt project for duck db, if really needed.

Reply:
Hi Tatiana,  we can reuse the same jaffle shop in the dev directory. I've tested with the example duckdb dag present.